tests/auto_generated/test_simplicial_complex_2d_hard.cpp

```diff
@@ -0,0 +1,4528 @@
+
+    #include <catch2/catch_test_macros.hpp>
+    #include <wmtk/SimplicialComplex.hpp>
+    #include <wmtk/TetMesh.hpp>
+    #include <wmtk/TriMesh.hpp>
+    #include "../tools/DEBUG_TriMesh.hpp"
+    #include <igl/read_triangle_mesh.h>
+
+    using namespace wmtk;
+
+    constexpr PrimitiveType PV = PrimitiveType::Vertex;
+    constexpr PrimitiveType PE = PrimitiveType::Edge;
+    constexpr PrimitiveType PF = PrimitiveType::Face;
+    constexpr PrimitiveType PT = PrimitiveType::Tetrahedron;
+    TEST_CASE("open_star_circle_0", "[simplicial_complex][open_star][2D][auto_generated]")
+{
+    RowVectors3d V;
+    RowVectors3l F;
+    std::string name = "/circle_0.7.obj";
```

Comment:
I don't think that file is in the data repo

Reply:
this belongs in the data repo - separate your code for auto-generating meshes and for validating meshes into two parts:
* generation goes in teh data repo + some generated meshes
* test generation lives in `tests/auto_generated/`